### PR TITLE
Function used to schedule Files workflow 

### PR DESCRIPTION
**Pull request by BergsethCognite**
# Description

Please describe the change you have made.

## Checklist

- [ ] Tests added/updated.
- [ ] Run Demo Job Locally.
- [X] Documentation updated.
- [ ] Changelogs updated in [CHANGELOG.cdf-tk.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.cdf-tk.md).
- [X] Template changelogs updated in [CHANGELOG.templates.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.templates.md).
- [ ] Version bumped.
  [_version.py](https://github.com/cognitedata/cdf-project-templates/blob/main/cognite/cognite_toolkit/_version.py) and
  [pyproject.toml](https://github.com/cognitedata/cdf-project-templates/blob/main/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by ronpal — APPROVED** [review]

File: CHANGELOG.templates.md [hunk 1]
```diff
@@ -26,11 +26,13 @@ Changes are grouped as follows:
...
 ### Added
 
-- Added Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
+- Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
+- Function used to schedule & trigger workflow
   
 ### Changed
 
 - The `cdf_functions_dummy` module now includes codes from the former `common_function_code` directory.
+- Removed schedule from annotation function `context:files:oid:fileshare:annotation`
```
Comment (doctrino) [inline CHANGELOG.templates.md:35]: Move up, `0.2.0a4` has been released.

**Review by doctrino — CHANGES_REQUESTED** [review]

File: CHANGELOG.templates.md (hunk 1 above)
Comment (BergsethCognite) [reply CHANGELOG.templates.md]: Done

File: CHANGELOG.templates.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows:
 
 ## TBD
 
+### Added
+
+- Function used to schedule & trigger workflow
```
Comment (doctrino) [inline CHANGELOG.templates.md:22]:
```suggestion
- In ` cdf_data_pipeline_files_valhall` example, added function used to schedule & trigger workflow.

```

File: CHANGELOG.templates.md [hunk 3]
```diff
@@ -26,11 +30,12 @@ Changes are grouped as follows:
 
 ### Added
 
-- Added Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
+- Workflow with a Function and a Transformation to the cdf_data_pipeline_files_valhall example
   
 ### Changed
 
 - The `cdf_functions_dummy` module now includes codes from the former `common_function_code` directory.
+- Removed schedule from annotation function `context:files:oid:fileshare:annotation`
```
Comment (doctrino) [inline CHANGELOG.templates.md:38]: Was this done earlier? Or was it done in this PR?

**Review by doctrino — CHANGES_REQUESTED** [review]
Need answer to question.

File: CHANGELOG.templates.md (hunk 3 above)
Comment (BergsethCognite) [reply CHANGELOG.templates.md:38]: this PR

**Review by doctrino — APPROVED** [review]

**Pull request merged**